scripts/updatekinto.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+const DEBUG = true;
+
+const got = require("got");
+
+const SERVER = "https://settings-writer.prod.mozaws.net/v1";
+const CID = "fxmonitor-breaches";
+/* eslint-disable-next-line no-process-env */
+const USERNAME = process.env.KINTO_USERNAME;
+/* eslint-disable-next-line no-process-env */
+const PASSWORD = process.env.KINTO_PASSWORD;
+if (!USERNAME || !PASSWORD) {
+  console.error("Please set credentials in the environment.");
+  return;
```

Comment:
Possibly add a non-zero exit code here? `process.exitCode = 1;`. Not that I imagine we'll run this script in CI or anything, but if it gets run on some sort of cron or OPs script, it may be nice knowing it hasn't been silently failing.